syntax = "proto3";

message DataMessage {
  int32 id = 1;
  string data = 2;
}

message RetransmitRequest {
  repeated int32 missing_ids = 1;
}